style.css

```diff
@@ -630,3 +647,430 @@
     color: #fff;
   }
 }
+
+@media screen and (min-width: 768px) {
+  *,
+  *::after,
+  *::before {
+    list-style: none;
+  }
+
+  body {
+    background-color: rgb(247, 248, 250);
+    z-index: -1;
+    height: 100%;
+  }
+
+  .header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 1rem 1.2rem 0 1.2rem;
+    height: 3rem;
+  }
+
+  .logo {
+    color: #6070ff;
+    font-size: var(--font-size-logo);
+    font-family: var(--font-family);
+    font-weight: 900;
+    text-decoration: none;
+    margin-left: 8vw;
+  }
+
+  .menu-img {
+    display: none;
+  }
+
+  .desktop-menu ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    margin-right: 8vw;
+    margin-top: -0.2rem;
+  }
+
+  .flex-3 li {
+    font-size: 20px;
+    font-family: var(--font-family);
+    color: #344563;
+  }
+
+  .desktop-menu ul li {
+    margin: 0 1rem 0 1rem;
+    font-size: 15px;
+    font-family: var(--font-family);
+    color: #344563;
+  }
+
+  #first-sec {
+    padding: 7rem 14vw 9rem 14vw;
+    background-image: url("images/Header\ bg@2x.png");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    height: 28rem;
+    border-bottom-left-radius: 100px;
+    background-color: #fff;
+    margin-bottom: 4rem;
+  }
+
+  #first-sec .mobile-h1 {
+    display: none;
+  }
+
+  .contact-form h1 {
+    font-family: var(--font-family);
+    font-size: var(--font-size-h2);
+    color: #fff;
+    display: flex;
+    justify-content: center;
+  }
+
+  .flex .flex-box h1 {
+    font-family: var(--font-family);
+    font-size: var(--font-size-h2-desktop);
+    color: #091e42;
+  }
+
+  #first-sec h1 {
+    margin-top: 3rem;
+    padding-top: 4rem;
+    font-size: var(--font-size-h1-desktop);
+    font-family: var(--font-family);
+    font-weight: 700;
+    letter-spacing: -3px;
+    color: #172b4d;
+  }
+
+  .contact-form p {
+    font-family: var(--font-family);
+    font-size: 20px;
+    color: #ebebff;
+    width: 40vw;
+    margin: -1rem 8rem 0 8rem;
+    text-align: justify;
+  }
+
+  .desktop-work-sec .div-info p {
+    font-family: var(--font-family);
+    font-size: var(--font-size-p);
+    color: #344563;
+    margin-left: 1.2rem;
+  }
+
+  .flex .flex-box p {
+    font-size: 16px;
+    font-family: var(--font-family);
+    color: #344563;
+  }
+
+  #first-sec p {
+    font-size: var(--font-size-p-desktop);
+    font-family: var(--font-family);
+    text-align: justify;
+    padding-top: -1rem;
+    margin-top: -1rem;
+  }
+
+  .flex .flex-box h3 {
+    font-family: var(--font-family);
+    font-size: 16px;
+    color: #7f8cff;
+  }
```

Comment:
- [ ] Please 🙏 reduce the gap 🔙 between each `about us` element to make it look like [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z0umLtHjW2jvzRBK-0) 


**_Figma Design_**

<img width="281" alt="Screenshot 2023-02-23 at 9 40 03 PM" src="https://user-images.githubusercontent.com/44798044/220981775-08102319-5473-484b-b75a-e47926a96f82.png">


**_Your Design_**

<img width="417" alt="Screenshot 2023-02-23 at 9 39 57 PM" src="https://user-images.githubusercontent.com/44798044/220981985-3801f4f5-4807-4fc6-955d-c2eb7fbb95a8.png">